rightmesh/contracts/MeshCrowdsale.sol

```diff
@@ -122,10 +122,27 @@ contract MeshCrowdsale is CappedCrowdsale, Ownable {
    * @return boolean indicating function success.
    */
   function setRate(uint256 _rate) external onlyOwner returns (bool) {
+    // make sure the crowdsale has not started
+    require(weiRaised == 0);
+
     rate = _rate;
     return true;
   }
 
+
+  /**
+   * @dev Allows the current owner to change the crowdsale cap.
+   * @param _cap indicating the new crowdsale cap.
+   * @return boolean indicating function success.
+   */
+  function setCap(uint256 _cap) external onlyOwner returns (bool) {
```

Comment:
This looks fine, but, if RM is going to be modifying the cap, what cap will they use when they launch the crowdsale contract? I believe this needs to be non-zero. ?

Reply:
what if cap here is higher than cap in the MeshToken contract?